packages/itwin/tree-widget/src/components/trees/common/FocusedInstancesContext.ts

```diff
@@ -7,9 +7,12 @@ import { createContext, useContext } from "react";
 
 import type { InstanceKey } from "@itwin/presentation-shared";
 import type { GroupingHierarchyNode } from "@itwin/presentation-hierarchies";
+import type { ClassGroupingNodeKey } from "@itwin/presentation-hierarchies/lib/cjs/hierarchies/HierarchyNodeKey";
+
+export type ClassGroupingHierarchyNode = GroupingHierarchyNode & { key: ClassGroupingNodeKey };
 
 export interface FocusedInstancesContext {
-  loadInstanceKeys?: () => AsyncIterableIterator<InstanceKey | GroupingHierarchyNode>;
+  loadFocusedItems?: () => AsyncIterableIterator<InstanceKey | ClassGroupingHierarchyNode>;
```

Comment:
I wanted to move this to `models-tree` folder but this would require additional changes in other trees. In the end, I've decided to revert this change